src/reducers/versions.tsx

```diff
@@ -322,6 +329,9 @@ export type VersionsState = {
     [compareInfoKey: string]: boolean;
   };
   currentVersionId: number | undefined | false;
+  entryStatusMap: {
```

Comment:
I suggest calling this `entryStatusMaps` to make it clear that it holds multiple maps.